inception-scheduling/src/main/java/de/tudarmstadt/ukp/inception/scheduling/SchedulingService.java

```diff
@@ -93,7 +110,12 @@ public synchronized void enqueue(Task aTask)
         }
 
         log.debug("Enqueuing task [{}]", aTask);
+        distributeWebSocketMessage(new TaskUpdateEvent(aTask, aTask.getUser().getUsername(),
```

Comment:
Events are normally published through an ApplicationEventPiublisher that you can have injected into your Spring component. Cf. other places where we publish Spring events.